### PR TITLE
Add avatar and Logout button

### PR DESCRIPTION
**Pull request by aashutoshrathi**

### CODE REVIEW

**Comment by yashLadha** [comment]
Avatar icon size can be reduced to accompany with the icon on its side. @aashutoshrathi . Else all looks good.

**Comment by aashutoshrathi** [comment]
On it

**Comment by aashutoshrathi** [comment]
screen

File: lib/screens/events.dart [hunk 1]
```diff
@@ -53,16 +53,30 @@ class _EventsPageState extends State<EventsPage> {
...
-            ),
+          Row(
+            children: <Widget>[
+              GestureDetector(
+                child: CircleAvatar(
+                  backgroundImage: CachedNetworkImageProvider(curUser.imgURL),
+                ),
+                onTap: () {
+                  // TODO: Implement Hero animation to go to Profile page
+                  print('Go to Profile');
```
Comment (yashLadha) [inline lib/screens/events.dart:64]: I also thought of the same, but what good will the profile page do?

Comment (aashutoshrathi) [reply lib/screens/events.dart:64]: It's like Dashboard on Website.

**Review by yashLadha — APPROVED** [review]
Lets Work On It, WIP

**Pull request merged**